vrecord

```diff
@@ -533,7 +533,7 @@ split=5[a][b][c][d][e];\
 [c1]${WAVEFORM_FILTER}[c2];\
 [a][b2][c2]vstack=inputs=3,format=yuv422p[abc1];\
 [d]${VECTORSCOPE_FILTER}[d1];\
-[e]format=yuv444p,lutyuv=y=if(gte(val\,254)\,1\,if(lte(val\,1)\,254\,val)):u=val:v=val,scale=512:ih[e1];\
+[e]format=yuv444p,"pseudocolor="'if(between(1,val,amax)+between(val,254,amax),65,-1):if(between(1,val,amax)+between(val,254,amax),100,-1):if(between(1,val,amax)+between(val,254,amax),212,-1)'"",scale=512:ih[e1];\
```

Comment:
The double-quotes aren't needed here, you're already within a double-quote starting on line 528. Are you sure this works without escaping the commas?

Reply:
I kept trying various combinations of quoting to escape the commas, and this *was* the only arrangement that kept visual from crashing on launch. Now suddenly it's not working again, so I will check back in tomorrow and try to see what's different from the line I had working earlier.

Reply:
FWIW, that chain works as part of a filter chain on my end with getting rid of all the double quotes and leaving the singles (which I think is the functional equivalent of `""'`...`'""`?)- so 

`pseudocolor='if(between(1,val,amax)+between(val,254,amax),65,-1):if(between(1,val,amax)+between(val,254,amax),100,-1):if(between(1,val,amax)+between(val,254,amax),212,-1)'`

also, killing all the quotes and escaping every comma with `\` seems to do the trick

Reply:
confirmed, no quotes at all + escaping commas works. got thrown off since the example for pseudocolor in the ffmpeg documentation had double + single quotes, but that must have something to do with the gradient function that doesn't matter here? fixing this line, anyway!